include/MySQL_Monitor.hpp

```diff
@@ -0,0 +1,81 @@
+#ifndef __CLASS_MYSQL_MONITOR_H
+#define __CLASS_MYSQL_MONITOR_H
+//#include <thread>
+//#include <list>
+//#include "btree_map.h"
+#include "proxysql.h"
+#include "cpp.h"
+
+#define MONITOR_SQLITE_TABLE_MYSQL_SERVER_CONNECT "CREATE TABLE mysql_server_connect (\
+  hostname VARCHAR NOT NULL,\
+  port INT NOT NULL DEFAULT 3306,\
+  time_since INT NOT NULL DEFAULT 0,\
+  time_until INT NOT NULL DEFAULT 0,\
+  connect_success_count INT NOT NULL DEFAULT 0,\
+  connect_success_first INT NOT NULL DEFAULT 0,\
+  connect_success_last INT NOT NULL DEFAULT 0,\
+	connect_success_time_min INT NOT NULL DEFAULT 0,\
```

Comment:
Just as a future observation, any metric's usefulness can be improved with additional metrics like AVG, STDDEV, percentile values.